doc/source/examples/extended_examples/Krylov/krylov_example.rst

```diff
@@ -243,28 +248,24 @@ Get the first 10 natural frequency modes of the acoustic duct
 
 Run harmonic analysis using Krylov method
 -----------------------------------------
+Perform the following steps to run the harmonic analysis using the
+frequency-sweep Krylov methd.
 
-**Step 1**: Generate full file.
+**Step 1**: Generate FULL file and initialize the ``Krylov`` class object.
 
 .. code:: ipython3
 
     mapdl.run('/SOLU')
-    mapdl.antype('HARMIC')  # HARMONIC ANALYSIS
+    mapdl.antype('HARMIC')  # Set options for harmonic analysis
     mapdl.hropt('KRYLOV')
     mapdl.eqslv('SPARSE')
-    mapdl.harfrq(0,1000)   # set beginning and ending frequency
-    mapdl.nsubst(100)      # set the number of frequency increments
-    mapdl.wrfull(1)        # GENERATE .FULL FILE AND STOP
+    mapdl.harfrq(0,1000)    # Set beginning and ending frequency
+    mapdl.nsubst(100)       # Set the number of frequency increments
+    mapdl.wrfull(1)         # Generate FULL file and stop
     mapdl.solve()
     mapdl.finish()
 
-
-
-Initialize Krylov class object.
-
-.. code:: ipython3
-
-    dd = mapdl.krylov
+    dd = mapdl.krylov       # Initialize Krylov class object
```

Comment:
I was trying to make all the code in a step one block--but this might not be the approach that you want to take. It just seemed here you used comments within the code block and having two lines separated seemed odd. However, I did not make such changes for subsequent steps.